ui/dashboards/src/components/TimeRangeControls/TimeRangeControls.test.tsx

```diff
@@ -37,7 +37,7 @@ describe('TimeRangeControls', () => {
   const renderTimeRangeControls = () => {
     renderWithContext(
       <DashboardProvider initialState={initialState}>
-        <TimeRangeProvider timeRange={testDefaultTimeRange}>
+        <TimeRangeProvider initialTimeRange={testDefaultTimeRange} paramsEnabled={false}>
```

Comment:
Maybe a bit more descriptive for the prop? `enabledURLParams` (or `urlParamsEnabled` in this case)?

Reply:
Maybe at some point, if it's a library - users might want to map the URL query params to avoid collision but we can probably punt on that for now until we figure out the public framework APIs

```tsx
<TimeRangeProvider urlParamsOptions={{ start: "from",  end: "until", prefix: "dashboard_"  }}  />
```